scripts/globals/spells/white/arise.lua

```diff
@@ -0,0 +1,32 @@
+-----------------------------------
+-- Spell: Arise
+-----------------------------------
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------
+local spell_object = {}
+
+spell_object.onMagicCastingCheck = function(caster, target, spell)
+    return 0
+end
+
+spell_object.onSpellCast = function(caster, target, spell)
+    if (target:isPC()) then
+        target:sendRaise(3)
+        target:setLocalVar("Arise", 1)
+    else
+        if (target:getName() == "Prishe") then
+            -- CoP 8-4 Prishe
+            target:setLocalVar("Raise", 1)
+            target:entityAnimationPacket("sp00")
+            target:addHP(target:getMaxHP())
+            target:addMP(target:getMaxMP())
```

Comment:
Wouldnt his block also work with Prishe, the trust?

Reply:
Shouldn’t be a problem, trust disappear as soon as they die, never giving a player a chance to raise them, this has been in every raise script previously and was added in this one for consistency

Reply:
somewhere in the future we should consider putting some kind of hook to the raise function to perform the work instead of duplicate scripting but even if trusts didn't despawn non players also require an target flag to even have raise cast on them and to my knowledge only that specific version of Prishe has it set out of all our entities in the database.